src/serac/physics/integrators/CMakeLists.txt

```diff
@@ -6,17 +6,21 @@
 
 set(integrators_sources
     hyperelastic_traction_integrator.cpp
-    inc_hyperelastic_integrator.cpp
+    displacement_hyperelastic_integrator.cpp
     wrapper_integrator.cpp
     )
 
 set(integrators_headers
     hyperelastic_traction_integrator.hpp
-    inc_hyperelastic_integrator.hpp
+    displacement_hyperelastic_integrator.hpp
     wrapper_integrator.hpp
     )
 
-set(integrators_depends serac_infrastructure)
+set(integrators_depends
+    serac_infrastructure
+    serac_numerics
+    serac_materials
+    )
 
 blt_add_library(
     NAME        serac_integrators
```

Comment:
To be consistent we should rename this `serac_physics_integrators` like we have named `serac_physics_utilities`